blindsidenetworks/scalelite
# SL-275: Removed hardcoded SHA256 checksum from BBB calls

fixes #905 
## Description

By default, Scalelite was hardcoded to always use SHA256 when making API calls to BBB. The environment variable `LOADBALANCER_CHECKSUM_ALGORITHM` allowed system admins to specify which algorithms can be used to verify the Scalelite checksum, but not the BBB checksum.

1. If not set, Scalelite will accept SHA1, SHA256, SHA512 requests and use SHA256 for requests to BBB
2. If set, Scalelite will accept whatever values are set, and use the most secure option for requests to BBB

## Testing Steps

1- Enabled SHA512 on your BBB server
2- Set `LOADBALANCER_CHECKSUM_ALGORITHM=SHA512`
3- Send a create call to Scalelite, using a SHA512 checksum
4- Ensure the request goes through without failing

## Screenshots (if appropriate):
